signing_service/signing_service/utils.py

```diff
@@ -16,6 +17,7 @@
 from golem_messages.exceptions import InvalidKeys
 
 from signing_service.constants import ETHEREUM_PRIVATE_KEY_REGEXP
+from signing_service.exceptions import SigningServiceValidationError
```

Comment:
I think it would be good to add a separate exception class for that case, as this was meant to be used to validate values inside SigningService.

Reply:
`InvalidValueError`, `Base64DecodeError` ?

Reply:
`Base64DecodeError` sounds good, `InvalidValueError` is too broad I think.